backends/aerospike.go

```diff
@@ -50,22 +50,29 @@ func (a *Aerospike) Get(ctx context.Context, key string) (string, error) {
 	return rec.Bins[binValue].(string), nil
 }
 
-func (a *Aerospike) Put(ctx context.Context, key string, value string, ttlSeconds int) error {
-	asKey, err := as.NewKey(a.cfg.Namespace, setName, key)
-	if err != nil {
-		return err
-	}
-	if ttlSeconds == 0 {
-		ttlSeconds = a.cfg.DefaultTTL
-	}
-	bins := as.BinMap{
-		binValue: value,
-	}
-	err = a.client.Put(&as.WritePolicy{
-		Expiration: uint32(ttlSeconds),
-	}, asKey, bins)
-	if err != nil {
-		return err
+func (a *Aerospike) MultiPut(ctx context.Context, payloads []Payload) error {
```

Comment:
Would it be beneficial to maintain existing behavior by keeping `Put` around, passing in an array with a single payload to `MultiPut`?